webpack.config.js

```diff
@@ -23,6 +23,7 @@ Encore
     .addEntry('app', './assets/app.js')
 
     .addEntry('home', './assets/pages/home.js')
+    .addEntry('single', './assets/pages/single.js')
```

Comment:
Codacy has a fix for the issue: [Replace `····.addEntry('single',·'./assets/pages/single.js'` with `··.addEntry("single",·"./assets/pages/single.js"`](https://app.codacy.com/gh/acecconato/AnthonyCecconato_5_18102021/pullRequest?prid=9466078)
```suggestion
  .addEntry("single", "./assets/pages/single.js")
```